kusto/cloudinfo.go

```diff
@@ -0,0 +1,138 @@
+package kusto
```

Comment:
This seems like a worthy addition, but there are some changes that need to be made to bring it up to the level it needs to be at.   There is a lot here to unpack here and I imagine it will take several go throughs to get to an end product.